src/Menu/Filters/GateFilter.php

```diff
@@ -25,6 +25,10 @@ public function transform($item, Builder $builder)
 
     protected function isVisible($item)
     {
+        if (isset($item['model'])) {
+            return ! isset($item['can']) || $this->gate->allows($item['can'], $item['model']);
+        }
+        
```

Comment:
Please remove the 8 spaces at this empty line.

Reply:
Sorry, I'm a bit new to this. I originally only saw this and pushed a change.